content/features/featuresDeepDive/physics/v2/forces.md

```diff
@@ -43,7 +43,13 @@ A force is a continuous effect that is applied to an object over time, which can
 
 An impulse, on the other hand, is a sudden, instantaneous effect that changes the velocity of an object. It is a specific amount of force applied over a very short duration of time, often modeled as a single frame in a game. For example, a collision between two objects might generate an impulse that changes the direction and speed of both objects.
 
-In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object.
+In summary, a force is a *continuous* effect over time, while an impulse is a sudden, *instantaneous* effect that changes the velocity of an object.  
+
+<Playground id="#R66K4K" title="applyForce" description="Show how to use applyForce in a very simple example" isMain={true} category="Physics"/>
+
+<Playground id="#Z8HTUN#671" title="applyImpulse" description="Show how to use applyImpulse in a very simple example" isMain={true} category="Physics"/>

```

Comment:
isMain={true} is a flag we are adding to specific playgrounds and not all. Would be great if you could remove it :-)

```suggestion
<Playground id="#R66K4K" title="applyForce" description="An example of using applyForce" category="Physics"/>

<Playground id="#Z8HTUN#671" title="applyImpulse" description="An example of using applyImpule" category="Physics"/>
```